### PR TITLE
Fix for build and deploy of 3360

### PR DESCRIPTION
**Pull request by jbrelsf2-nmdp**
Modified just enough to get this working.    3360 has two IDs that appear to be missing a ".". These two
IDs are removed from the file before processing.

The two sequences are:
C*05:208N
C*05:206

Current use of the mhalagannmdp docker image is intentional.  

ToDo: I noticed that the pygfe and pyGFE also might need to be updated.    The Docker build  process will need to be unpacked so that he dockerfile is the recipie for the build for that entire image from an already supported public Biopython image.

The current method is not very transparent and we are relying on the image that was created previously.  This will not always work and will also have eventual vulnerabilities.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,15 @@
-FROM mhalagan1nmdp/gfe-base:latest
+FROM mhalagan1nmdp/gfe-base:latest 
 
-COPY bin/* opt/
-COPY requirements.txt opt/
+ADD bin/* /opt/
```
Comment (pbashyal-nmdp) [inline Dockerfile]: Why the change to `ADD` ? `ADD` is only for URLs or gzip files. See Docker Best Practices: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

File: bin/build_gfedb.py [hunk 2]
```diff
@@ -625,6 +632,7 @@ def main():
             if hasattr(allele, 'seq'):
                 hla_name = allele.description.split(",")[0]
                 loc = allele.description.split(",")[0].split("*")[0]
+                print (hla_name, skip_alleles, loc)
```
Comment (pbashyal-nmdp) [inline bin/build_gfedb.py]: Is this needed ? Too much of `print` will just add more noise to logs.

File: bin/build_gfedb.py [hunk 3]
```diff
@@ -90,7 +95,9 @@ def hla_alignments(dbversion):
         sth_prot = data_dir + "/../data/" + dbversion + "/" + loc.split("-")[1] + "_prot.sth"
 
         logging.info("Loading " + sth_gen)
-        align_gen = AlignIO.read(open(sth_gen), "stockholm")
+        print(skip_alleles)
+        #align_gen = AlignIO.read(open(sth_gen), "stockholm")
+        align_gen = next(AlignIO.parse(sth_gen, "stockholm"))
```
Comment (pbashyal-nmdp) [inline bin/build_gfedb.py]: Why the change to `parse` ?

Comment (jbrelsf2-nmdp) [reply bin/build_gfedb.py]: This has been moved back to AlignIO.read.  This was the wrong direction in resolving this issue.

File: bin/build_gfedb.py (hunk 2 above)
Comment (jbrelsf2-nmdp) [reply bin/build_gfedb.py]: I will remove this print.  i agree.  Too messy.

Comment (jbrelsf2-nmdp) [reply bin/build_gfedb.py]: Any debug print info has been removed.

File: Dockerfile (hunk 1 above)
Comment (jbrelsf2-nmdp) [reply Dockerfile]: moved back to COPY where appropriate.
